scripts/source-maintenance.sh

```diff
@@ -45,9 +53,11 @@ printUsage () {
     echo "    --keep-going|-k"
     echo "    --check-and-update-copyright <yes|no>"
     echo "    --with-astyle </path/to/astyle/executable>"
+    echo "    --changed-only"
 }
 
 # command-line options
+CHANGEDONLY=""
```

Comment:
> CamelCase with shell scripts looks extremely odd.

They do not look odd at all to me.

> Most conventions rely on capitalized or underscore-separated variable names

I do not know whether that assertion is actually true -- the very [first](https://stackoverflow.com/questions/673055/correct-bash-and-shell-script-variable-capitalization) [two](https://unix.stackexchange.com/questions/42847/are-there-naming-conventions-for-variables-in-shell-scripts) references I found say otherwise:

> By convention, environment variables (PAGER, EDITOR, ...) and internal shell variables (SHELL, BASH_VERSION, ...) are capitalized. All other variable names should be lower case.

... both then explain why using all-caps script variables is a bad idea.

What I do know is that it is actually difficult for me to read the new variable names in this PR, and I am sure I am not alone. I also know that this script already uses CamelCase (among other styles) so I am not asking for anything new here (not to mention other Squid sources).

Reply:
TBH I don't care enough to resist the change request.  I will retain externally-input variables (CHECKSUM, STYLE) in uppercase for similarity with PAGER, EDITOR in your advice

Reply:
done in d5b6cb5dc

Reply:
> I will retain externally-input variables (CHECKSUM, STYLE) in uppercase

As you know, I think those should not exist at all (or replaced with a single `SQUID_SOURCE_MAINTENANCE_OPTIONS` variable), but that is the focus of another change request.